common/src/chain/transaction.rs

```diff
@@ -103,4 +103,18 @@ impl Transaction {
             Transaction::V1(tx) => tx.get_serialized_hash(),
         }
     }
+
+    pub fn is_coinbase(&self) -> bool {
```

Comment:
We won't have coinbase transactions. Coinbase will be embedded in the header with Consensus data.

Reply:
Ohh, cool! Just today thought to write you about the coinbase. Then I'm going to remove it.